### PR TITLE
New Structure to the website

### PR DESCRIPTION
**Pull request by wrussell1999**
This isn't polished, but a good first start at reorganising the website.

I've split the website into several sections:

- Events
    - Hack Soc Lists
- Attend
    - Workshops
- Organise
    - Before The Event
    - During The Event
    - After The Event
    - Help From The Community
- Sponsor
- Volunteer
- Art

The `Sponsor` page can be turned into section, as proposed in [Issue 42](https://github.com/HHEU/wiki/issues/42). 

The `Help From The Community` is a start at looking at how HHEU can directly help hackathons, as well as having hackathons help each other, with the idea that HHEU creates a platform for them to do that very easily. Also, this looks at how HHEU can have dedicated people (Coaches/Mentors) for hackathons, as well as people who have good backgrounds in certain areas such as Catering, Venue, Sponsorship, AV.

The `Organise` section contains new pages within the `Before The Event`, `During The Event`, and `After The Event`. A lot of these are not complete, but contributions would be appreciated.

I would love to hear your feedback on this before it is approved and merged!

### CODE REVIEW

**Comment by aaronosher** [comment]
this is where having branch deployments using [Heroku review-apps](https://devcenter.heroku.com/articles/github-integration-review-apps) would be very nice

**Comment by wrussell1999** [comment]
> this is where having branch deployments using [Heroku review-apps](https://devcenter.heroku.com/articles/github-integration-review-apps) would be very nice

You can view my branch [here](https://deploy-preview-77--jolly-shaw-db0aa5.netlify.com/)

**Comment by jmsv** [comment]
:+1: using netlify rather than heroku, so we've got preview builds on PR by default :sparkles:

@wrussell1999 there's a few `TODO`s — are these supposed to be resolved before merge, or would they perhaps be better as comments?

**Comment by wrussell1999** [comment]
@jmsv it would be ideal if they could be resolved before the merge. I should've created a draft PR in reality.

Contributions to the TODOs would be great. Some of them are probably fine to leave in case someone else can contribute. Using VSCode Search should find them all.

**Comment by wrussell1999** [comment]
@jmsv it would be ideal if they could be resolved before the merge. I should've created a draft PR in reality.

Contributions to the TODOs would be great. Some of them are probably fine to leave in case someone else can contribute. Using VSCode Search should find them all.

Tomorrow I'm going to fill in some more. Pages that have not a lot of content should be commented out for the time being.

**Comment by wrussell1999** [comment]
@jmsv I’ve removed more TODOs now. Only ones remaining are for:

- Volunteers, During The Event (I’ll sort this later today)
- Mentors, During The Event (I’ll sort this later today)
- Judging, During The Event (Help here would be appreciated)

**Comment by bahorn** [comment]
Following warnings in the build:

```
5:44:16 AM: INFO    -  The following pages exist in the docs directory, but are not included in the "nav" configuration:
5:44:16 AM:   - art/providers.md
5:44:16 AM:   - art/hexbin/readme.md
5:44:16 AM:   - organise/before/health_and_safety.md
5:44:16 AM: WARNING -  A relative path to 'organise/before/heath_and_safety.md' is included in the 'nav' configuration, which is not found in the documentation files
5:44:16 AM: WARNING -  A relative path to 'organise/help/community.md' is included in the 'nav' configuration, which is not found in the documentation files
5:44:16 AM: WARNING -  Documentation file 'attend/workshops.md' contains a link to 'attend/IntroToGit.md' which is not found in the documentation files.
```

The ones lacking links in the navbar don't matter too much beyond the health and safety, which is just a typo.

I gather community hasn't been done yet, so it'd be worth putting a TODO there.

Last one just needs a quick change to fix the path.

**Comment by wrussell1999** [comment]
New TODO added to `/organise/help/community.md`. Intention is for a list of people who can help in specific areas like sponsorship, catering, venue, safeguarding, AV, etc...

Something @robzwolf and I talked about, but worth discussing with the others.

**Comment by bahorn** [comment]
Probably worth moving the content on the `art/providers` page into `organise/before/stash_and_swag`. It'l clear up the last of the warnings and we can ignore the last one anyway.

I can get it done a bit later.

**Comment by bahorn** [comment]
Just merged the changes i mentioned this morning. Might need some quick grammar corrections

**Comment by wrussell1999** [comment]
Volunteer and mentor TODOs are now complete. Just the judging page to do before everything is ready.

File: docs/organise/help/poi.md [hunk 1]
```diff
@@ -125,4 +125,31 @@ Updated 2020 Season
...
+
+* [Tom Goodman](https://twitter.com/TauOmicronMu)
+* [Liam Sorta](https://twitter.com/LiamSorta)
+* [Helena Bower](https://twitter.com/bowerhe)
+* [Will Russell](https://twitter.com/wrussell1999)
+* [Charlie de Freitas](https://twitter.com/chasbob97)
+* [Jon Kingsley](https://twitter.com/JFKingsley)
+* [Justin Chadwell](https://twitter.com/jedevc)
+* [Lik Kan Chung](https://twitter.com/LikkanChung)
+* [Daniel Spencer](mailto:danielfspencer@gmail.com)
```
Comment (danielfspencer) [inline docs/organise/help/poi.md]: danielfspencer1@gmail.com

**Comment by wrussell1999** [comment]
I have hidden the Judging page until it gets completed @jmsv. I have also added some basic buttons on the home page to make it easier to find things on the website. I have some future ideas I will share, once I've noted them, about how the home page can be improved to cater for both the community and wiki aspect of hack.athon.uk.

**Comment by wrussell1999** [comment]
Some styling tweaks on the home page buttons would be helpful (not that most experience web dev). Adding a margin between the logo and the buttons, as well as making all the buttons a fixed width.

**Comment by bahorn** [comment]
Screenshot from 2019-12-16 04-24-27

Had it render like this on various different screen resolutions.

`mkdocs` does support actually adding it to the top bar, you can see examples on how it looks at these sites:

* https://squidfunk.github.io/mkdocs-material/
* https://wiki.makerdiary.com/

**Comment by bahorn** [comment]
So when enabling tabs, it ends up like this:

Screenshot from 2019-12-16 04-54-08

So, we'll have to probably rework the order of sections to only get the four or five main ones we care about in the top bar.

So:
* Attend
* Community
* Organise
* Sponsor
* Volunteer

I'd put art under community, put the old events page under attend and maybe move sponsor and volunteer under like a help section?

**Comment by wrussell1999** [comment]
That looks much better. Yeah that cleans it up. Just swap Volunteer with Help!

**Comment by wrussell1999** [comment]
Are these social links correct:

```
extra:
  social:
    - type: globe
      link: https://hackathonhackers.eu
    - type: facebook
      link: https://www.facebook.com/groups/hackathonhackerseu/
    - type: twitter
      link: https://twitter.com/_HHEU
    - type: github-alt
      link: https://github.com/hheu
```

**Comment by danielfspencer** [comment]
I'm liking the new structure. Can we move the Past Events table off the front page? Maybe make another page for it?

**Comment by wrussell1999** [comment]
Yup plan is to put this into the attend section I believe

**Comment by robzwolf** [comment]
> Yup plan is to put this into the attend section I believe
> […](#)

We should definitely try and keep the list of upcoming events front and centre on the homepage @wrussell1999, tbh I’d be in favour of making them **more** visible than they currently are. 

Could put the past events on a separate page and leave the upcoming ones on the homepage?

**Comment by wrussell1999** [comment]
That's what I mean, when referring to Dan's comment 😂

**Comment by wrussell1999** [comment]
When making the upcoming events better looking, we should look into an easier way to add them, and iCal stuff (all previously mentioned somewhere)

File: docs/organise/during/judging.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Judging
+
+## Who should be your judges?
+
+TODO:
```
Comment (jmsv) [inline docs/organise/during/judging.md]: Are these done?

**Comment by wrussell1999** [comment]
That looks better. If you could push the changes, that would be great.

As mentioned in a previous comment, I've commented out the judging page
from the menu. I've asked Jack (or anyone with experience running judging)
to write it.

For now, we can merge without the judging page if all okay?

**Comment by wrussell1999** [comment]
Also, the buttons seem to have weird behaviour on some screen sizes. Could
you have a look into that please?

**Comment by jmsv** [comment]
on it :100:

**Comment by bahorn** [comment]
Removing the `community.md` page from the navbar and removed the TODOs from judging.

Should be ready for merging now?

**Comment by wrussell1999** [comment]
Yup should be fine. Those other changes can be smaller PRs

**Comment by bahorn** [comment]
and that ends the 9 day saga!

**Pull request merged**